architecture/additional_concepts/sdn.adoc

```diff
@@ -109,7 +111,7 @@ network footnote:[After this point, device names refer to devices on container
 B's host.] -> vxlan0 -> br0 -> vovsbr -> vlinuxbr -> lbr0 -> vethB -> eth0 (in
 B's netns)_
 
-== External Access to the Cluster Network
+==== External Access to the Cluster Network
```

Comment:
Is this only applicable to the ovssubnet plug-in, as its current placement suggests?